docs/tasks/debug-application-cluster/audit.md

```diff
@@ -70,3 +147,105 @@ the file name (before the file extension) and create a new audit log file.
 Kubernetes may delete old log files when creating a new log file; you can configure
 how many files are retained and how old they can be by specifying the `audit-log-maxbackup`
 and `audit-log-maxage` options.
+
+Logged events are printed as:
```

Comment:
It's probably worth pointing out that the audit log format does not include request or response, so `Request` and `RequestResponse` levels are equivalent to `Metadata` with this backend.